### PR TITLE
github: Add ISSUE_TEMPLATE.md and PULL_REQUEST_TEMPLATE.md

### LINKED ISSUE #250: Add an Issue and Pull Request template on Github for the community

### PR DESCRIPTION
**Pull request by html5cat**
Fixes #250

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+**Do you want to request a *feature* or report a *bug*?**
```
Comment (leodasvacas) [inline .github/ISSUE_TEMPLATE.md]: Or ask a question.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+**What is the current behavior?**
+
+**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
+
+**What is the expected behavior?**
+
+**Which OS, version of Rust, PostgreSQL and IPFS are you using?**
```
Comment (leodasvacas) [inline .github/ISSUE_TEMPLATE.md]: This seems excessive, most bugs will be independent from the version of these dependencies. What do you think of removing this and just ask when relevant?

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (html5cat) [reply .github/ISSUE_TEMPLATE.md]: If it's a question probably should go to Discord?

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (html5cat) [reply .github/ISSUE_TEMPLATE.md]: Sure!

**Comment by html5cat** [comment]
@leodasvacas added placeholder for now – what do you think should go to PR template?

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (leodasvacas) [reply .github/ISSUE_TEMPLATE.md]: Ideally, but if GitHub is their favorite mean of communication then they should feel welcome to post questions.

**Comment by leodasvacas** [comment]
Having a placeholder seems fine, not sure what to put on it, perhaps after we get some experience with contributors we'll get ideas for that.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (html5cat) [reply .github/ISSUE_TEMPLATE.md]: I'd rather not encourage that 😄

**Pull request merged**